monty/exts/info/docs/_cog.py

```diff
@@ -660,10 +660,8 @@ def maybe_defer(inter: disnake.Interaction) -> None:
             if not no_match:
                 if isinstance(inter, disnake.Interaction):
                     self.bot.loop.call_later(2, maybe_defer, inter)
-                elif hasattr(inter, "trigger_typing"):
+                else:
```

Comment:
why was this reverted?

Reply:
This is the 1st commit, as I said I pulled your commit a few minutes ago